scheduled-jobs/build/oc-mirror-quay-dev/Jenkinsfile

```
@@ -1,68 +1,58 @@
+node {
+    checkout scm
 
+    def commonlib = load( "pipeline-scripts/commonlib.groovy")
```

```suggestion
    def commonlib = load("pipeline-scripts/commonlib.groovy")
```

---

Also
```suggestion
    def commonlib = buildlib.commonlib
```
it's the same but ISTR that it treats the two loads as two separate scripts, depending on load order, which makes a replay really confusing

---

Not going to keep updating the old scheduled-jobs file.